cookbook/localization/how_to_use_the_presenters.rst

```diff
@@ -0,0 +1,42 @@
+How to use Presenters
+=====================
+
+The Presenters are used to display the data for the end user according to their local or other parameters.
```

Comment:
@skeleton Here we only use presenters to present data from a specific locale ; the structure of the presenter can be used to have any option (anything)